packages/react-components/src/components/RichTextEditor/Plugins/KeyboardInputPlugin.tsx

```diff
@@ -1,23 +1,41 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import type { PluginEvent, EditorPlugin } from 'roosterjs-content-model-types';
+import type { PluginEvent, EditorPlugin, IEditor } from 'roosterjs-content-model-types';
 import { PluginEventType } from '../../utils/RichTextEditorUtils';
 
 /**
  * KeyboardInputPlugin is a plugin for handling keyboard events in the editor.
  */
 export class KeyboardInputPlugin implements EditorPlugin {
+  private editor: IEditor | null = null;
+  private disposer: (() => void) | null = null;
```

Comment:
Why do we need to use null here instead of undefined?

Reply:
I think it can be either value. I think initially I followed a pattern from RoosterJS (or thre were some dependencies on RoosterJS plugins) and that's why we use null there. 
I can update it, not a strong preference

Reply:
Agreed to review this later when there is a reason not to use null here because of the required testing and refactoring for all plugins